content/en/containers/amazon_ecs/apm.md

```diff
@@ -139,6 +146,17 @@ For more examples of setting the Agent hostname in other languages, see the [cha
 
 {{< programming-lang lang="nodeJS" >}}
 
+#### Launch time variable
+Update the Task Definition's `entryPoint` with:
+```json
+"entryPoint": [
+  "sh",
+  "-c",
+  "export DD_AGENT_HOST=$(curl http://169.254.169.254/latest/meta-data/local-ipv4); npm start"
```

Comment:
See comment for Python, but same idea. Not all NodeJS applications run with `npm start`.

```
"export DD_AGENT_HOST=$(curl http://169.254.169.254/latest/meta-data/local-ipv4); <command used to launch NodeJS application container>"
```